### PR TITLE
add UI for the homepage 

### LINKED ISSUE #34: UI: PHIN VADS landing page
Create a `templ` template for the PHIN VADS home page to match the CDC styles (based on Heather's [mockup](https://www.figma.com/design/2qu1dogZOKe4qXW1QEhEJh/PHIN-VADS-Prototype?node-id=41-1585&t=CejJahZOAqO0hPxg-0) and the CDC [home page](https://www.cdc.gov/))
- create a new route that's note `/home` where this will live

### PR DESCRIPTION
**Pull request by akintner**
Resolves #34 

- adds search bar
- adds preliminary Hot Topic (incomplete for now) dropdowns
- adds styling for above elements

### CODE REVIEW

**Comment by akintner** [comment]
Alright, this is up for basic functionality (nothing being pulled from database yet, so placeholder text and functionality with search).

File: internal/ui/components/home.templ [hunk 1]
```diff
@@ -3,8 +3,98 @@ package components
...
+							src="/assets/img/usa-icons-bg/search--white.svg"
+							class="usa-search__submit-icon"
+							alt="Search"
+						/>
+						</button>
+					</form>
+				</section>
+			</div>
+			<div class="grid-row">
+				<div class="grid-col-8 grid-offset-8">
```
Comment (katyasoup) [inline internal/ui/components/home.templ:40]: can get rid of this div, and just have the `<p>` nested directly under the `grid-row` div; if you then add a style to that remaining div (maybe add a unique class name), and set `justify-contents: flex-end`, the Advanced Search button will stay all the way over to the right! (Might need to throw some extra margin px in there to make it line up perfectly)

File: internal/ui/assets/css/styles.css [hunk 2]
```diff
@@ -122,9 +122,37 @@ header.cdc-header .header-search {
...
 
-.vh {
-  height: 50vh;
-  background-color: orange;
+.usa-accordion__button {
+  margin-bottom: 12px;
+}
+
+.usa-button {
+  height: 3rem;
```
Comment (katyasoup) [inline internal/ui/assets/css/styles.css:130]: can add `max-width: 20rem` to make this match the size of the dropdown on the left side

File: internal/ui/assets/css/styles.css [hunk 3]
```diff
@@ -122,9 +122,37 @@ header.cdc-header .header-search {
...
+}
+
+.usa-input {
```
Comment (katyasoup) [inline internal/ui/assets/css/styles.css:133]: if you make this `max-width: 100%` and adjust the button as noted above, this should stretch everything to cover the full width of the container

**Review by katyasoup — COMMENTED** [review]
Looks great! I noted a few places where you could tweak things to make the search portion at the top stretch across the whole width of the content; otherwise, just a couple styles that we get from USWDS that can be removed from the css file:

`.usa-input` (the height property here is also causing the header's input to grow taller than the button)
`.usa-search__input` - I don't see anywhere in our code using this class name, we can remove it
`.usa-search usa-search--big` and `.usa-button` styles also come from USWDS

File: internal/ui/components/home.templ (hunk 1 above)
Comment (akintner) [reply internal/ui/components/home.templ:40]: took some tweaking with the other search fields, but I got it fixed eventually in a way that didn't go wonky if you get down to smaller window sizes.

**Review by katyasoup — APPROVED** [review]
🎉

**Pull request merged**